src/integration/pages/articles/afrique/amp.test.js

```diff
@@ -4,9 +4,15 @@
  */
 
 import runGistTests from '../gistTests';
+import runMediaPlayerCaptionTests from '../mediaPlayerCaption';
+import runMediaPlayerPlaceholderTests from '../mediaPlayerPlaceholderTestsCanonical';
```

Comment:
Did we mean to run a canonical test in the AMP tests? I think we need to include the mediaPlayerPlaceholderTestsAmp tests instead?

```suggestion
import runMediaPlayerPlaceholderTests from '../mediaPlayerPlaceholderTestsAmp';
```